showmethekey-gtk/meson.build

```diff
@@ -51,7 +51,7 @@ executable(
   dependencies: gtk_dependencies,
   include_directories: gtk_include_directories,
   install: true,
-  install_dir: get_option('prefix') / 'bin'
+  install_dir: get_option('prefix') / get_option('bindir')
```

Comment:
Here too.

Reply:
Fixed in ff967cd4832b0685260c1a66ad9a3442a8cb7b26.